docs/design.md

```diff
@@ -138,7 +138,7 @@ The compactor also does additional batch processing such as down-sampling and ap
 
 ## Scaling
 
-None of the Thanos components provides any means of sharding. The only explicitly scalable component are query nodes, which are stateless and can be scaled up arbitrarily. Scaling of storage capacity is ensured by relying on an external object storage system.
+None of the Thanos components provides any means of sharding. The only explicitly scalable component are query nodes, which are stateless and can be scaled out arbitrarily. Scaling of storage capacity is ensured by relying on an external object storage system.
```

Comment:
> None of the Thanos components provides any means of sharding.

Wow, this does not hold truth anymore ;p 

I think at this point we need to rename it as `Initial Design`? (:
cc @brancz